DataFormats/HGCalReco/interface/HGCalSoACellsOut.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DataFormats_HGCalReco_interface_HGCalSoACellsOut_h
+#define DataFormats_HGCalReco_interface_HGCalSoACellsOut_h
+
+#include "DataFormats/SoATemplate/interface/SoACommon.h"
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/SoATemplate/interface/SoAView.h"
+
+// SoA layout with delta, rho, weight, nearestHigher, clusterIndex, layer, isSeed, and cellsCount fields
+GENERATE_SOA_LAYOUT(HGCalSoACellsOutLayout,
```

Comment:
```suggestion
GENERATE_SOA_LAYOUT(HGCalSoARecHitsExtraLayout,
```
or
```suggestion
GENERATE_SOA_LAYOUT(HGCalSoARecHitsForClusteringLayout,
```
or something else more meaningful ?